routes/web.php

```diff
@@ -14,5 +16,25 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
-});
+    return view('/posts');});
+
+
+//Resource Controller
+Route::resource('posts', PostController::class);
+
+//Routegroup
+Route::group(['prefix'=> 'Posts.', 'as' => 'POSTS.'], function(){
+
+    Route::post('/create-post', [PostController::class, 'store']) ->name('store');
```

Comment:
Please change '/create-post' to just '/create'.
No need to add 'posts' since we already have it in the URL.
Please apply to others below with '-post'.